tests/map_js_test.go

```diff
@@ -0,0 +1,117 @@
+//go:build js && !wasm
+// +build js,!wasm
+
+package tests
+
+import (
+	"github.com/gopherjs/gopherjs/js"
+	"testing"
+)
+
+func Test_MapWrapper(t *testing.T) {
+	// This tests that various map types, and a map as a function argument and return,
+	// wrap and unwrap correctly.
+	type Dummy struct {
+		Msg string
+	}
+
+	type StructWithMap struct {
+		StringMap map[string]string
+		IntMap    map[int]int
+		DummyMap  map[string]*Dummy
+		MapFunc   func(map[string]string) map[string]string
+	}
+
+	dummyMap := map[string]*Dummy{`key`: {Msg: `value`}}
```

Comment:
nit: I notice you are using backticks for all string literals. For consistency with the rest of the codebase it would be better to use double-quotes for single-line literals.